compiler_opt/tools/make_corpus_lib.py

```diff
@@ -33,7 +33,7 @@ def load_bitcode_from_directory(bitcode_base_dir: str) -> List[str]:
   Returns an array of paths representing the relative path to the bitcode
   file from the base direcotry.
   """
-  paths = [str(p) for p in pathlib.Path(bitcode_base_dir).glob('**/*.bc')]
+  paths = [str(p)[:-3] for p in pathlib.Path(bitcode_base_dir).glob('**/*.bc')]
```

Comment:
nit: to make it more clear where that "3" came from, you can just have something like:
```
extension = '.bc'
... str(p)[:-len(extension)]
```

and maybe also the glob can be ('**/*' + extension)